packages/dapp-simple-exchange/contract/deploy.js

```diff
@@ -125,17 +74,28 @@ export default async function deployContract(homeP, { bundleSource, pathResolve
   // === AWAITING TURN ===
   // =====================
 
-  console.log('- installation made', CONTRACT_NAME, '=>',  installationHandle);
+  console.log('- installation made', CONTRACT_NAME, '=>',  contractId);
   console.log('- instance made', CONTRACT_NAME, '=>', instanceId);
 
-    // Save the instanceId somewhere where the UI can find it.
+  // Save the instanceId somewhere where the UI can find it.
   if (instanceId) {
-    const cjfile = pathResolve(`../ui/src/utils/contractID.js`);
-    console.log('writing', cjfile);
-    await fs.promises.writeFile(cjfile, `export default ${JSON.stringify(instanceId)};`);
-
-    // =====================
-    // === AWAITING TURN ===
-    // =====================
+    const dappConstants = {
+      BRIDGE_URL: 'agoric-lookup:https://local.agoric.com?append=/bridge',
+      API_URL: '/',
+      CONTRACT_ID: instanceId,
+    };
+    const dc = 'dappConstants.js';
+    console.log('writing', dc);
+    await fs.promises.writeFile(dc, `globalThis.__DAPP_CONSTANTS__ = ${JSON.stringify(dappConstants, undefined, 2)}`);
+
+    // Now add URLs so that development functions without internet access.
```

Comment:
Now add URLS so that [what] works without internet?

Reply:
Comment updated.